explorer/client/src/utils/static/mock_data.json

```diff
@@ -181,6 +181,12 @@
                 {
                     "objectId": "standaloneObject"
                 },
+                {
+                    "objectId": "standaloneObject"
+                },
+                {
+                    "objectId": "standaloneObject"
```

Comment:
multiple objects with the same object ID isn't possible in the network - can we change this mock data ?

Reply:
Yes and done